crates/sqlbuiltins/src/functions/scalars.rs

```diff
@@ -0,0 +1,870 @@
+);
+make_const!(
+    ISZERO,
+    name => "iszero",
+    example => "iszero(0)",
+    description => "Returns true if the argument is zero, false otherwise."
+);
+make_const!(
+    LCM,
+    name => "lcm",
+    example => "lcm(10, 15)",
+    description => "Compute the least common multiple of two integers."
+);
+make_const!(
+    LN,
+    name => "ln",
+    example => "ln(1)",
+    description => "Compute the natural logarithm of a number."
+);
+make_const!(
+    LOG,
+    name => "log",
+    example => "log(1)",
+    description => "Compute the logarithm of a number in a specified base."
+);
+make_const!(
+    LOG10,
+    name => "log10",
+    example => "log10(1)",
+    description => "Compute the base-10 logarithm of a number."
+);
+make_const!(
+    LOG2,
+    name => "log2",
+    example => "log2(1)",
+    description => "Compute the base-2 logarithm of a number."
+);
+make_const!(
+    NANVL,
+    name => "nanvl",
+    example => "nanvl(1, 2)",
+    description => "Returns the first argument if it is not NaN, otherwise returns the second argument."
+);
+make_const!(
+    PI,
+    name => "pi",
+    example => "pi()",
+    description => "Compute the constant π."
+);
+
+make_const!(
+    POWER,
+    name => "pow",
+    example => "pow(2, 3)",
+    description => "Compute the power of a number."
+);
+make_const!(
+    RADIANS,
+    name => "radians",
+    example => "radians(1)",
+    description => "Converts an angle measured in degrees to an approximately equivalent angle measured in radians."
+);
+make_const!(
+    RANDOM,
+    name => "random",
+    example => "random()",
+    description => "Compute a random number between 0 and 1."
+);
+make_const!(
+    ROUND,
+    name => "round",
+    example => "round(1.1)",
+    description => "Round a number to the nearest integer."
+);
+make_const!(
+    SIGNUM,
+    name => "signum",
+    example => "signum(1)",
+    description => "Compute the sign of a number."
+);
+make_const!(
+    SIN,
+    name => "sin",
+    example => "sin(0)",
+    description => "Compute the sine of a number."
+);
+make_const!(
+    SINH,
+    name => "sinh",
+    example => "sinh(0)",
+    description => "Compute the hyperbolic sine of a number."
+);
+make_const!(
+    SQRT,
+    name => "sqrt",
+    example => "sqrt(4)",
+    description => "Compute the square root of a number."
+);
+make_const!(
+    TAN,
+    name => "tan",
+    example => "tan(0)",
+    description => "Compute the tangent of a number."
+);
+make_const!(
+    TANH,
+    name => "tanh",
+    example => "tanh(0)",
+    description => "Compute the hyperbolic tangent of a number."
+);
+make_const!(
+    TRUNC,
+    name => "trunc",
+    example => "trunc(1.1111, 2)",
+    description => "Truncate a number to the nearest integer. If a second argument is provided, truncate to the specified number of decimal places."
+);
+make_const!(
+    COALESCE,
+    name => "coalesce",
+    example => "coalesce(null, 1)",
```

Comment:
is the first argument a list, since this is a scalar function and not a window function?

Reply:
it's as the description says. Returns the first non null **argument**. 
- `coalesce(1,2,3,4,5,6,7, null, 8) -> 1`
- `coalesce(null, 1) -> 1`
- `coalesce(null, null, null, 1) -> 1`